test/Tests/src/Semantic/Conversion_Use.enso

```diff
@@ -0,0 +1,9 @@
+from Standard.Base import all
+
+type Hello
+    Say (msg:Text)
+
+    formulate : Vector Hello -> Text
+    formulate arr =
+        process (t:Text) (h:Hello) = t + h.msg
```

Comment:
By using ascribed arguments in a function that `fold`s over the `Vector`, we request each element to be `Hello` or converted to `Hello`.